src/feature/wines/detail/components/WineFlavorProfileSection.tsx

```
@@ -0,0 +1,75 @@
+    const tasteMappings = {
+      바디감: 'lightBold',
+      타닌: 'smoothTannic',
+      당도: 'drySweet',
+      산미: 'softAcidic',
+    } as const;
+
+    const averages = Object.entries(tasteMappings).reduce(
+      (acc, [tasteName, reviewKey]) => ({
+        ...acc,
+        [tasteName]:
+          reviews.reduce((sum, review) => sum + review[reviewKey], 0) /
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)